### ISSUE TITLE
Front broken after react-ui merge

### ISSUE THREAD

**Issue by samip5** [DangerOnTheRanger/maniwani#165]
The front seems to be broken when building the dev container and using that.

The  file at `/static/css/stock/theme-stock.css` does not exist.

Build also partially fails:

```
> maniwani@0.0.1 gulp /maniwani
> gulp

[11:51:39] Using gulpfile /maniwani/Gulpfile.js
[11:51:39] Starting 'default'...
[11:51:39] Starting 'clean'...
[11:51:39] Finished 'clean' after 8.56 ms
[11:51:39] Starting 'fontawesome_webfonts'...
[11:51:39] Starting 'maniwani_css'...
[11:51:39] Starting 'fontawesome_css'...
[11:51:39] Starting 'popper'...
[11:51:39] Starting 'jquery'...
[11:51:39] Starting 'bootstrap_js'...
[11:51:39] Starting 'imagesloaded'...
[11:51:39] Starting 'masonry'...
[11:51:39] Finished 'masonry' after 36 ms
[11:51:39] Finished 'imagesloaded' after 38 ms
[11:51:39] Finished 'popper' after 39 ms
[11:51:39] Finished 'fontawesome_css' after 40 ms
[11:51:39] Finished 'bootstrap_js' after 41 ms
[11:51:39] Finished 'jquery' after 42 ms
[11:51:39] Finished 'fontawesome_webfonts' after 48 ms
Error in plugin "sass"
Message:
    scss/themes/harajuku/theme-harajuku.scss
Error: File to import not found or unreadable: board-index.
        on line 10 of scss/themes/harajuku/theme-harajuku.scss
>> @import "board-index";

   ^

[11:51:39] Finished 'maniwani_css' after 75 ms
[11:51:39] Finished 'default' after 89 ms
Removing intermediate container 0d17b5c4dacb
 ---> cfddfaf84ccf
Step 18/41 : WORKDIR /maniwani-frontend
 ---> Running in e359ac31dc37
Removing intermediate container e359ac31dc37
 ---> 5cf042c78687
Step 19/41 : COPY frontend/package.json /maniwani-frontend
 ---> b5305f5cd02c
Step 20/41 : COPY frontend/package-lock.json /maniwani-frontend
 ---> 5be6433c4818
Step 21/41 : RUN npm install
 ---> Running in 18d329170cd2
npm WARN maniwani-frontend@1.0.0 No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 842 packages from 350 contributors and audited 912 packages in 7.966s

3 packages are looking for funding
  run `npm fund` for details

found 348 vulnerabilities (285 low, 63 high)
  run `npm audit fix` to fix them, or `npm audit` for details
Removing intermediate container 18d329170cd2
 ---> 4ef33fec9eaf
```

![image](https://user-images.githubusercontent.com/1703002/95198455-f4257c00-07e3-11eb-8f0f-20c32e564342.png)

**Comment by DangerOnTheRanger** [comment]
I forgot to commit `board-index.scss`, probably due to trying to push the PR through at 3AM, oops. Should be fixed as of f8f850c6a1efb7d43589ee227caa8f2f70e9fc1f.

**Comment by samip5** [comment]
Seems to be fine now.
